### PR TITLE
qa/standalone/osd/pg-split-merge: fix import_after_merge_and_gap

### PR DESCRIPTION
**Pull request by liewegas**
Fixes: http://tracker.ceph.com/issues/38525

### CODE REVIEW

File: src/osd/OSD.cc [hunk 1]
```diff
@@ -6433,6 +6440,9 @@ COMMAND("cache drop",
 COMMAND("cache status",
         "Get OSD caches statistics",
         "osd", "r")
+COMMAND("send_beacon",
+        "Second OSD beacon to mon immediately",
```
Comment (neha-ojha) [inline src/osd/OSD.cc]: nit: s/Second/Send/

**Comment by neha-ojha** [comment]
jenkins retest docs

**Comment by liewegas** [comment]
http://pulpito.ceph.com/sage-2019-03-02_01:13:07-rados-wip-sage2-testing-2019-03-01-1553-distro-basic-smithi/

File: src/osd/OSD.cc (hunk 1 above)
Comment (liewegas) [reply src/osd/OSD.cc]: fixed!

**Pull request merged**